workflows/trana.nf

```diff
@@ -118,13 +127,13 @@ workflow TRANA {
         //
         if (!params.skip_cutadapt) {
             CUTADAPT(ch_reads)
```

Comment:
I'm a little confused why CUTADAPT doesn't process `ch_processed_passed_reads`, or at least `ch_processed_reads` ... is this correct? It seems it will otherwise bypass the filtering?

Reply:
If you change `skip_cutadapt` from `false` to `true` in `nextflow.config` then `CUTADAPT` should run. I haven't changed anything regarding that.